src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/dto/ActionPlanSessionsDTO.kt

```diff
@@ -7,12 +7,20 @@ import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.Attende
 import java.time.OffsetDateTime
 import java.util.UUID
 
+data class AddressDTO(
+  val firstAddressLine: String,
+  val secondAddressLine: String? = "",
+  val townOrCity: String,
+  val county: String,
```

Comment:
_nit_ Just a random observation: addresses usually don't require `county`

Reply:
I've made both county and townOrCity optional now